Worktez/worktez
# fix(teams-card-ui):"Fixed the Ui for teams card and created new UI for team details"

### Functionality:
fixing UI for teams card and create new UI team details page

### Solution:
fixed the UI for the teams card and create a new UI for the teams page

### Risk level:

### How to test:
step 1: check the team cards in my organization.
step 2: check the team details page from the team cards
